src/main/java/org/sonarlint/intellij/ui/CurrentFileConnectedModePanel.java

```diff
@@ -79,24 +86,57 @@ private void createPanel() {
     layout = new CardLayout();
     panel = new JPanel(layout);
 
+    var notConnectedCardWrapper = new JPanel(new GridBagLayout());
+    var connectedCardWrapper = new JPanel(new GridBagLayout());
+
+    var helpLabelConnected = HelpLabelUtils.createCurrentFileHelp();
+    var helpLabelNotConnected = HelpLabelUtils.createCurrentFileHelpNotConnected();
+
+    var gce = new GridBagConstraints(GridBagConstraints.EAST, 0, 1, 1, 0, 0, GridBagConstraints.EAST, GridBagConstraints.NONE,
+      JBUI.insets(2, 20, 2, 0), 0, 0);
+    var gcs = new GridBagConstraints(GridBagConstraints.CENTER, 0, 1, 1, 0, 0, GridBagConstraints.CENTER, GridBagConstraints.NONE,
+      JBUI.insets(2, 5, 2, 0), 0, 0);
+
+    var helpLabelTextConnected = new JLabel("What's in this view");
+    var helpLabelTextNotConnected = new JLabel("What's in this view");
+
     connectedCard = new ConnectedCard();
+    notConnectedCard = new NotConnectedCard();
+    emptyCard = new EmptyCard();
+    errorCard = new ErrorCard();
+
+    addMouseClickListeners();
+
+    notConnectedCardWrapper.add(helpLabelTextNotConnected);
+    notConnectedCardWrapper.add(helpLabelNotConnected, gcs);
+    notConnectedCardWrapper.add(notConnectedCard, gce);
 
-    panel.add(new EmptyCard(), EMPTY);
-    panel.add(new NotConnectedCard(), NOT_CONNECTED);
-    panel.add(connectedCard, CONNECTED);
-    panel.add(new ErrorCard(), ERROR);
+    connectedCardWrapper.add(helpLabelTextConnected);
+    connectedCardWrapper.add(helpLabelConnected, gcs);
+    connectedCardWrapper.add(connectedCard, gce);
```

Comment:
In fact, I wonder if we shouldn't extract the logic for the "What's in this view" in a different panel. One effect is that if no file is opened, the "empty" card gets selected, so the "What's in this view" disappears. We can discuss that online if you want